Realm/Realm.Fody/Realm.Fody.csproj

```diff
@@ -16,7 +16,6 @@
   </PropertyGroup>
   <ItemGroup Label="References">
     <PackageReference Include="FodyHelpers" Version="6.*" PrivateAssets="All" />
-    <PackageReference Include="System.Management" Version="4.5.0" PrivateAssets="Build;Compile;Runtime" />
```

Comment:
This I suppose is something completely unrelated or does it have to do with sleet?

Reply:
It is indeed unrelated. It's just that when I was testing the sleet nuget feed, I noticed that we have a single dependency in the weaver package, which seemed odd and it turned out to be something we don't really need.